pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -67,6 +73,7 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 					pods, err := i.coreClient.Pods(namespace).List(metav1.ListOptions{})
 					if err != nil {
 						klog.V(2).Infof("Unable to find pods in namespace %s for failing operator %s", namespace, item.Name)
+						continue
```

Comment:
the `pods` can be nil on error and this will then panic...

Reply:
good catch